source/heatfluxClass.py

```diff
@@ -554,6 +555,34 @@ def multiExp_profile_fluxspace(self, PFC, R, Bp, psiN, mode):
         q[nan_locations] = 0.0
         q[inf_locations] = 0.0
         return q
+    
+    def from_rzq_profile(self, rzq_data, PFC):
+        """"
+        q from any R(m), Z(m), q(W/m2) data
+        q is mapped from the R,Z grid to OMP
+        """
+
+        R = rzq_data['R(m)'].to_numpy()
+        Z = rzq_data['Z(m)'].to_numpy()
+        Q = rzq_data['q(W/m2)'].to_numpy()
+        # make Q positive. Sometimes, the q values from SOLPS is negative to indicate direction.
+        for i in range(0, len(Q)):
```

Comment:
I suggest vectorizing this:
```suggestion
        negs = np.where(Q<0.0)[0]
        Q[negs] *= -1.0
```

Reply:
done